lib/tool/code_interpreter.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Tool
+  class CodeInterpreter < Base
+    description <<~DESC
+      Useful for executing ruby code.
```

Comment:
Is this the best description we could provide it? I'm curious if anyone else has done it and what kind of description have they used. Like should we tell it that the expected input to this tool is a valid Ruby code or expression? May need to be tested out. #promptengineering.

Reply:
Good point. How about this (based on LC):

```A Ruby code interpreter. Use this to execute ruby expressions. Input should be a valid ruby expression. If you want to see the output of the tool, make sure to return a value.```